trustme/__init__.py

```diff
@@ -350,6 +357,9 @@ def issue_server_cert(self, *hostnames):
                 chain_to_ca,
             )
 
+    # For backwards compatibility
+    issue_server_cert = issue_cert
```

Comment:
Thanks for keeping backwards compatibility! I don't know if that's worth the trouble without a deprecation warning, though.

Reply:
+1 for deprecation warning